public/assets/application-fbf25f87b816945058df04f4e29fe6317f0e0da6028c95c47944560f74bdf5a2.css

```diff
@@ -0,0 +1,299 @@
+/* line 1, /home/fabrizio/Documents/Sublime/Rails/railsmentor/app/assets/stylesheets/questions.scss */
```

Comment:
Bundled file. Also, personal information - since no one, except you, ever keeps files in `/home/fabrizio/Documents/Sublime/Rails/railsmentor`.